assets/js/base/components/cart-checkout/order-summary/order-summary-item.js

```diff
@@ -50,6 +51,15 @@ const OrderSummaryItem = ( { cartItem } ) => {
 		.multiply( quantity )
 		.convertPrecision( currency.minorUnit )
 		.getAmount();
+	const subtotalPriceFormat = __experimentalApplyCheckoutFilter( {
+		filterName: 'subtotalPriceFormat',
+		defaultValue: '',
+		arg: {
+			lineItem: cartItem,
+		},
+		// Only accept strings.
+		validation: ( value ) => typeof value === 'string',
```

Comment:
Should we check that it includes `<price/>` too?

Reply:
Good catch, thanks Albert